Src/Base/AMReX_GpuLaunch.H

```diff
@@ -215,6 +231,22 @@ namespace Gpu {
 #define AMREX_HOST_DEVICE_PARALLEL_FOR_3D(...) AMREX_GPU_HOST_DEVICE_PARALLEL_FOR_3D(__VA_ARGS__)
 #define AMREX_HOST_DEVICE_PARALLEL_FOR_4D(...) AMREX_GPU_HOST_DEVICE_PARALLEL_FOR_4D(__VA_ARGS__)
 
+#ifdef AMREX_USE_CUDA
+#define AMREX_HOST_DEVICE_PARALLEL_FOR_1D_FUSABLE(...) AMREX_GPU_HOST_DEVICE_FOR_1D_FUSABLE(__VA_ARGS__)
+#define AMREX_HOST_DEVICE_PARALLEL_FOR_3D_FUSABLE(...) AMREX_GPU_HOST_DEVICE_FOR_3D_FUSABLE(__VA_ARGS__)
+#define AMREX_HOST_DEVICE_PARALLEL_FOR_4D_FUSABLE(...) AMREX_GPU_HOST_DEVICE_FOR_4D_FUSABLE(__VA_ARGS__)
+#define AMREX_HOST_DEVICE_FOR_1D_FUSABLE(...) AMREX_GPU_HOST_DEVICE_FOR_1D_FUSABLE(__VA_ARGS__)
+#define AMREX_HOST_DEVICE_FOR_3D_FUSABLE(...) AMREX_GPU_HOST_DEVICE_FOR_3D_FUSABLE(__VA_ARGS__)
+#define AMREX_HOST_DEVICE_FOR_4D_FUSABLE(...) AMREX_GPU_HOST_DEVICE_FOR_4D_FUSABLE(__VA_ARGS__)
+#else
```

Comment:
Does this mean the "FUSABLE" macros exist to work around HIP and DPCPP and they will go away? (I'm a little concerned about the amount of complexity that's building in this.)

Reply:
It's just a hint.  Whether or not HIP and DPCPP support it, we can keep them.